inc/classes/class-imagify-user.php

```diff
@@ -152,6 +170,10 @@ public function __construct() {
 	public function get_percent_consumed_quota() {
 		static $done = false;
 
+		if ( $this->get_error() ) {
+			return 0;
+		}
+
```

Comment:
Does having an error mean that the process is not done, i.e. `$done -= false;`?

Reply:
`$done` is only used to prevent hooks to be triggered several times for nothing, so it has no meaning for the process itself.